src/pages/Checkout/CheckoutPage.vue

```diff
@@ -572,6 +581,9 @@ export default {
 				if (hasFreeCredits) {
 					if (refreshEvent === 'kiva-card-applied') {
 						this.$kvTrackEvent('basket', 'free credits applied', 'exit to legacy');
+						if (this.checkingOutAsGuest) {
+							this.disableGuestCheckout();
+						}
```

Comment:
I don't think we need the `if` here, I think we can just call `this.disableGuestCheckout()` because we've already checked that free credits were being applied to the basket.